src/layouts/index.js

```diff
@@ -34,14 +34,14 @@ const Layout = props => (
             cta: 'donate',
           }}
         />
-        <div>{props.children()}</div>
-      </div>
+        <main>{props.children()}</main>
```

Comment:
Wow! I had no idea there was a `main` tag in html! And I thought I'd seen it all! 😅